pallets/weights.rs

```diff
@@ -0,0 +1,57 @@
+
+//! Autogenerated weights for `pallet_randomness`
```

Comment:
Why this file exists and is in `pallets/weights.rs`, but it is for pallet randomness?

Reply:
fixed